__tests__/fixtures/list/dev-deps-prod/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "left-pad": "^1.1.3"
```

Comment:
Hmmm ... could you point me to a test that follows that pattern? I was basing most of my structure off of the `no-args` tests / fixtures